homeassistant/components/wyoming/stt.py

```diff
@@ -126,6 +126,9 @@ async def async_process_audio_stream(
                         text = transcript.text
                         break
 
+        except TimeoutError:
```

Comment:
Why was this added?

Reply:
Without it the call state wouldn't get properly cleaned up which I think was causing subsequent calls to not create a new RTP server and there would be no audio on the phone.

Reply:
I removed it in the latest change, I am still having issues with it but it is not the same as before so that might have been a red herring when I added it the first time. I think I need to increase the timeout value because it seems like when I say a command it will make the tone like it heard the command, but then shortly after it hangs up on me from what I assume is caused by the timeout error.

Reply:
I added this back in because hanging up after the regular command pipeline loop wouldn't work without it.